package/base-files/files/lib/upgrade/fwtool.sh

```diff
@@ -1,20 +1,14 @@
+CHECK_SYSUPGRADE_SIGNATURE="${CHECK_SYSUPGRADE_SIGNATURE:-$(uci get system.@system[-1].check_sysupgrade_signature)}"
+
 fwtool_check_signature() {
 	[ $# -gt 1 ] && return 1
 
-	[ ! -x /usr/bin/ucert ] && {
-		if [ "$REQUIRE_IMAGE_SIGNATURE" = 1 ]; then
-			return 1
-		else
-			return 0
-		fi
-	}
-
 	if ! fwtool -q -s /tmp/sysupgrade.ucert "$1"; then
 		echo "Image signature not found"
-		[ "$REQUIRE_IMAGE_SIGNATURE" = 1 -a "$FORCE" != 1 ] && {
```

Comment:
BTW what's the reason for renaming `REQUIRE_IMAGE_SIGNATURE` to `CHECK_SYSUPGRADE_SIGNATURE` ? I would simply keep it as it is and just add `require_image_signature` UCI option.

There is currently similar option, `REQUIRE_IMAGE_METADATA` and it allows to enable/disable the metadata checking per platform, so I would simply don't rename it in order to stay consistent. What do you think?

Reply:
I though the uci option would be more *verbose* after renaming, but I'm happy with both...

Reply:
Well you can set `REQUIRE_IMAGE_SIGNATURE` with `check_sysupgrade_signature` UCI option, but then it would be probably confusing. Anyway, it's probably just a matter of taste, but I simply prefer to not cause headaches to downstream projects more then necessary :-)